docs/fedora-livemedia.ks

```diff
@@ -389,4 +388,13 @@ syslinux
 -gfs2-utils
 -reiserfs-utils
 
+# This package is needed to boot the iso on UEFI
+shim
+shim-ia32
```

Comment:
I think this is actually `shim-*` since it's `shim-x64` and `shim-ia32`.

Reply:
shim is a magic package that installs shim-x64. i would say it's a meta package. but it doesn't come up in the dnf search. hence magic.